pkg/gedcom/warnings.go

```diff
@@ -0,0 +1,37 @@
+package gedcom
+
+const (
+	LevelDebug Level = iota - 1
+	LevelInfo
+	LevelWarning
```

Comment:
Still trying to think of a use case for Info and Debug levels. I guess for testing new functionality. I'm changing the tests that look for warnings to only consider `LevelWarning` or higher as failures.